.circleci/config.yml

```diff
@@ -0,0 +1,194 @@
+version: 2
+workflows:
+  version: 2
+  tests:
+    jobs: &workflow_jobs
+      - node6:
+          filters: &all_commits
+            tags:
+              only: /.*/
+      - node8:
+          filters: *all_commits
+      - node10:
+          filters: *all_commits
+      - lint:
+          requires:
+            - node6
+            - node8
+            - node10
+          filters: *all_commits
+      - docs:
+          requires:
+            - node6
+            - node8
+            - node10
+          filters: *all_commits
+      - system_tests:
+          requires:
+            - lint
+            - docs
+          filters: &master_and_releases
+            branches:
+              only: master
+            tags: &releases
+              only: '/^v[\d.]+$/'
+      - sample_tests:
+          requires:
+            - lint
+            - docs
+          filters: *master_and_releases
+      - publish_npm:
+          requires:
+            - system_tests
+            - sample_tests
+          filters:
+            branches:
+              ignore: /.*/
+            tags: *releases
+  nightly:
+    triggers:
+      - schedule:
+          cron: 0 7 * * *
+          filters:
+            branches:
+              only: master
+    jobs: *workflow_jobs
+jobs:
+  node6:
+    docker:
+      - image: 'node:6'
+        user: node
+    steps: &unit_tests_steps
+      - checkout
+      - run: &remove_package_lock
+          name: Remove package-lock.json if needed.
+          command: |
+            WORKFLOW_NAME=`python .circleci/get_workflow_name.py`
+            echo "Workflow name: $WORKFLOW_NAME"
+            if [ "$WORKFLOW_NAME" = "nightly" ]; then
+              echo "Nightly build detected, removing package-lock.json."
+              rm -f package-lock.json samples/package-lock.json
+            else
+              echo "Not a nightly build, skipping this step."
+            fi
+      - run: &npm_install_and_link
+          name: Install and link the module
+          command: |-
+            mkdir -p /home/node/.npm-global
+            npm install
+            npm link
```

Comment:
You can delete this line and the line below (the chmod).  Would you mind also updating the repo-tools script that generates this?

Reply:
Yeah, this whole template needs to be updated.  Where does this file come from?